shell/platform/darwin/ios/framework/Source/SemanticsObjectTest.mm

```diff
@@ -0,0 +1,40 @@
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+
+#include "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/SemanticsObject.h"
+
+using namespace flutter;
```

Comment:
Avoid `using namespace` https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
Done.  Checkout out how it says "never use using directives" then about 50 lines later they use one in an example.  Other google projects seem to allow them as long as they are inside of a scoped namespace.  I just removed it.

Reply:
lol. WDYT about having our own version of the guidelines? We ought to be able to edit or amend them for clarity.

Reply:
If we want to go that way we should make it an addendum that clarifies ambiguities like this as they show up.